src/clojureSuggest.ts

```diff
@@ -23,14 +23,13 @@ export class ClojureCompletionItemProvider implements vscode.CompletionItemProvi
         if (!cljConnection.isConnected())
             return Promise.reject('No nREPL connected.');
 
-        // TODO: Use VSCode means for getting a current word
-        let lineText = document.lineAt(position.line).text;
-        let words: string[] = lineText.split(' ');
-        let currentWord = words[words.length - 1].replace(/^[\('\[\{]+|[\)\]\}]+$/g, '');
-        let text = document.getText()
-        let ns = cljParser.getNamespace(text);
+        let wordRange = document.getWordRangeAtPosition(position);
```

Comment:
`wordRange` can be const.

Reply:
Thank you, I've fixed it and turned some next vars to `const` too.

Reply:
Sorry for the mistake in prev commit message `*some`, anyway it should be squashed)